components/org.wso2.carbon.identity.organization.management.service/src/test/java/org/wso2/carbon/identity/organization/management/service/dao/impl/OrganizationManagementDAOImplTest.java

```diff
@@ -0,0 +1,248 @@
+import java.sql.Timestamp;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.List;
+import java.util.TimeZone;
+
+import static java.time.ZoneOffset.UTC;
+import static org.wso2.carbon.identity.organization.management.service.constant.OrganizationManagementConstants.OrganizationTypes.STRUCTURAL;
+import static org.wso2.carbon.identity.organization.management.service.util.Utils.generateUniqueID;
+import static org.wso2.carbon.identity.organization.management.util.TestUtils.closeH2Base;
+import static org.wso2.carbon.identity.organization.management.util.TestUtils.getConnection;
+import static org.wso2.carbon.identity.organization.management.util.TestUtils.initiateH2Base;
+import static org.wso2.carbon.identity.organization.management.util.TestUtils.mockDataSource;
+
+public class OrganizationManagementDAOImplTest {
+
+    private OrganizationManagementDAO organizationManagementDAO = new OrganizationManagementDAOImpl();
+    private static final Calendar CALENDAR = Calendar.getInstance(TimeZone.getTimeZone(UTC));
+    private static final String ATTRIBUTE_KEY = "country";
+    private static final String ATTRIBUTE_VALUE = "Sri Lanka";
+    private static final String ORG_NAME = "XYZ builders";
+    private static final String INVALID_DATA = "invalid data";
+    private static final String SUPER_ORG_ID = "10084a8d-113f-4211-a0d5-efe36b082211";
+    private String orgId;
+
+    @BeforeClass
+    public void setUp() throws Exception {
+
+        initiateH2Base();
+        mockDataSource();
+        storeChildOrganization(SUPER_ORG_ID);
+    }
+
+    @AfterClass
+    public void tearDown() throws Exception {
+
+        closeH2Base();
+    }
+
+    @Test
+    public void testAddOrganization() throws Exception {
+
+        String orgId = generateUniqueID();
+
+        Organization organization = new Organization();
+        organization.setId(orgId);
+        organization.setName("org1");
+        organization.setDescription("org1 description.");
+        organization.setCreated(Instant.now());
+        organization.setLastModified(Instant.now());
+        organization.setStatus(OrganizationManagementConstants.OrganizationStatus.ACTIVE.toString());
+        organization.setType(STRUCTURAL.toString());
+
+        ParentOrganizationDO parentOrganizationDO = new ParentOrganizationDO();
+        parentOrganizationDO.setId(SUPER_ORG_ID);
+        organization.setParent(parentOrganizationDO);
+
+        List<OrganizationAttribute> attributes = new ArrayList<>();
+        attributes.add(new OrganizationAttribute(ATTRIBUTE_KEY, ATTRIBUTE_VALUE));
+        organization.setAttributes(attributes);
+
+        organizationManagementDAO.addOrganization(organization);
+        Assert.assertNotNull(organizationManagementDAO.getOrganization(orgId));
+    }
+
+    @DataProvider(name = "dataForIsOrganizationExistByName")
+    public Object[][] dataForIsOrganizationExistByName() {
+
+        return new Object[][]{
+
```

Comment:
shall we remove this unnecessary new line

Reply:
resolved by 6534bc4a65f7a5c7396abda24169fb0e5690702a